index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
+    <link rel="stylesheet" href="https://ssl.gstatic.com/docs/script/css/add-ons1.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  </head>
+  <body>
+    <div class = "sidebar">
+     <p>Click the Button to Check In and Out 
+     <p>
+     <br>
+     <input class="action" id="checkinout" type="button" value="" onmouseup="checkingInOut()">
+    </div>
+     
+    <script>
+    
+    // Initially Set the button status
+    setButtonStatus();
+    
+    /* Run the setButtonStatus to set the text on the button */              
+    function setButtonStatus() {
+    
+      console.log("In setButtonStatus")
+      var checkedIn = google.script.run.withSuccessHandler(setText).withFailureHandler(function(err){ alert("Error on isCheckedIn")}).isCheckedIn();       
```

Comment:
That works but is kinda clunky and you are not displaying the error. 

Add a hidden div with id="error" then set the value to the error message.
   
    <input ...
    <div id="error" hidden><!-- Error message will go here --></div>

    function errorHandler(error) { // Same one for all the "script.google.run"s
      document.getElementById("error").value = error.message
      document.getElementById("error").style.display = "inline" // "none" to hide it again
    }

Reply:
Yes that is clearer.